test/gameplay/cards/SorcerousSpellWall.js

```diff
@@ -0,0 +1,21 @@
+const assert = require('chai').assert;
```

Comment:
The test is nice and concise, covering what it needs to. You might not want to use hard coded variable for the attack value (2800), but rather constants or values from the classes being tested, If one of the values is changed in a future commit, any tests with a hard coded value will also have to be manually changed as well.